packages/mason_cli/lib/src/commands/make.dart

```diff
@@ -156,6 +159,11 @@ class _MakeCommand extends MasonCommand {
         );
       }
 
+      if (setExitIfChanged) {
+        final modified = target.dir.statSync().modified != lastModified;
+        if (modified) return ExitCode.software.code;
```

Comment:
Do we want to log something here to show why we're exiting? Just a nit

Reply:
What do you suggest? I was thinking that since you'd see the list of files generated it'd be clear which were identical and which have been modified but I'm open to adding an additional log.

Reply:
I was thinking something simple, like "Exiting due to conflicts."